app/controllers/groups_controller.rb

```diff
@@ -131,7 +131,7 @@ def authorize_admin_group!
   end
 
   def set_title
-    @title = 'New Group'
+    @title = I18n.t("gitlab.groups.new_group")
```

Comment:
Why not to use https://github.com/ruby-gettext/gettext ? it have "contexts" support built in.